test/shared/src/main/scala/zio/test/package.scala

```diff
@@ -50,7 +50,7 @@ import scala.language.implicitConversions
  * }}}
  */
 package object test extends CompileVariants {
-  type TestEnvironment = Annotations with Live with Sized with TestConfig
+  type TestEnvironment = Annotations with Live with Sized with TestConfig with TestConsole
```

Comment:
This change is not necessary.

Reply:
Good catch,  vestige from one of the earlier approaches. Removed.